item.rb

```diff
@@ -0,0 +1,23 @@
+class Item
+  def initialize(id, publish_date, archived)
+    @id = id
```

Comment:
## Optional 
 
Kindly consider giving a random id while initializing an id. You can do this by using the rand() method